### DISCUSSION TITLE
New LTMEAT works on GoGuardian, but what about Cisco Umbrella?

### DISCUSSION THREAD

**Discussion by Mazedotexe** [3kh0/ext-remover#729] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
cisco umbrella does not have a file urls switch
![Screenshot 2023-10-18 8 03 22 PM](https://github.com/3kh0/ext-remover/assets/108310732/7a39310b-cce8-4c95-b840-ae08c52e0039)

**Comment by Mazedotexe** [comment]
help lol

**Comment by AshtonDavies** [comment, marked as answer]
> cisco umbrella does not have a file urls switch
> ![Screenshot 2023-10-18 8 03 22 PM](https://github.com/3kh0/ext-remover/assets/108310732/7a39310b-cce8-4c95-b840-ae08c52e0039)
> 

The extension has webRequest permissions, but does not include the file URLs switch. You would need to find an obscure way to refresh your extension. The only other way to do this is by killing it, which is very possible, but too hard to plug into this method, sorry
